### PR TITLE
Added missing content and smaller changes to FRD_resultedservice.md

### PR DESCRIPTION
**Pull request by c21axepe**
I added the missing content and also made minor adjustments!

### CODE REVIEW

File: backend-models/functional-requirements/FRD_resultedservice.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
+| ------------- | ------------- |
+| FR1 | Teacher/super-user shall be able to view all student results |
+| FR2 | Teacher/super-user shall be able to filter student results |
 
 | FN No| Test Prosoal: |
-| FR1| Try to make a submission to see if it shows correct informations |
-| FR2| Try to filter a dugga |
+| ------------- | ------------- |
+| FR1| Login as different teachers/super-users and see if all can access the information |
+| FR2| Try to filter the student results |
```
Comment (b21ricni) [inline backend-models/functional-requirements/FRD_resultedservice.md:9]: Tests that is focused more on data and that correct values are sent. for example: 
"Filter the student results and compare the result with database result with the values suppose to show"

**Review by b21ricni — CHANGES_REQUESTED** [review]
Tests that is more focused on data and that correct values are sent

File: backend-models/functional-requirements/FRD_resultedservice.md (hunk 1 above)
Comment (c21axepe) [reply backend-models/functional-requirements/FRD_resultedservice.md]: The tests are now more data focused

**Review by b21ricni — APPROVED** [review]
Nice!

**Pull request merged**